pages/devs/starknetreact.mdx

```diff
@@ -156,3 +222,95 @@ function Component() {
   return <span>address: {data}</span>;
 }
 ```
+
+---
+
+#### useStarkProfile
```

Comment:
Can you add this first in the order ? We want people to use it as much as possible and the button example you're using it so it makes sense to make it first in the list